terraform/environments/ppud/alb.tf

```
@@ -0,0 +1,97 @@
+
+# PPUD ALB Configuration
+
+resource "aws_lb" "PPUD-ALB" {
+  count              = local.is-development == true ? 1 : 0
+  name               = "PPUD-ALB"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.PPUD-ALB.id]
+  subnets            = [data.aws_subnet.public_subnets_a.id, data.aws_subnet.public_subnets_b.id]
+
+  enable_deletion_protection = false
+
+  tags = {
+    Name = "${var.networking[0].business-unit}-${local.environment}"
+  }
+}
+
+resource "aws_lb_listener" "PPUD-Front-End" {
+  load_balancer_arn = aws_lb.PPUD-ALB[0].arn
+  port              = "443"
+  protocol          = "HTTPS"
+  ssl_policy        = "ELBSecurityPolicy-2016-08"
+  certificate_arn   = "arn:aws:acm:eu-west-2:075585660276:certificate/e45192bf-7779-4f83-808d-139381cb9c02"
```

```suggestion
  certificate_arn   = "arn:aws:acm:eu-west-2:${data.aws_caller_identity.current.account_id}:certificate/e45192bf-7779-4f83-808d-139381cb9c02"
```
Please don't hard code account numbers where possible, it makes it harder to rebuild the platform if we need to, you can reference the current account number like above

---

removed hard code account numbers